### ISSUE TITLE
Version on official website not updated

### ISSUE THREAD

**Issue by mwelsch** [bestiejs/benchmark.js#224]
Hey, the version on the website https://benchmarkjs.com/ still says 2.1.2 while the newest version is 2.1.4.
Cheers!

**Comment by austindd** [comment]
Indeed, the documentation there is not up to date, and the links from the website to the GitHub source code actually point to version 2.0.0 of the source code.
